Blackjack.Tests/ControllerShould.cs

```diff
@@ -40,9 +40,16 @@ public void ReturnDealerWin_GivenPlayerBust()
                 .Returns(new Card(CardRank.Two, CardSuit.Clubs))
                 .Returns(new Card(CardRank.Four, CardSuit.Diamonds));
 
-            controller.Play();
+            var playerHand = player.Hand;
+            var dealerHand = dealer.Hand;
+            var expectedGameResult = new GameResult(dealerHand, playerHand);
+
+            var expectedOutcome = Outcome.DealerWin;
+
+            var result = controller.Play();
 
-            Assert.Equal("Dealer wins!", output.GetWinner()); 
+            Assert.True(GameResultHelper.GameResultsAreEqual(expectedGameResult, result));
+            Assert.Equal(expectedOutcome, result.Outcome);

```

Comment:
I would say this kind of functionality should be tested as part of the `GameResult` tests. The behaviour that we want the `Controller` to exhibit is encapsulated in the `IOutput` i.e. we change `IInput` in some way (also mocking out the `Deck` to control for randomness) and expect the `IOutput` to change accordingly.